.taskcluster.yml

```diff
@@ -1,215 +1,181 @@
 # This Source Code Form is subject to the terms of the Mozilla Public
 # License, v. 2.0. If a copy of the MPL was not distributed with this
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
-
+---
 version: 1
+reporting: checks-v1
 policy:
-  pullRequests: public
+  # XXX We restrict taskcluster to collaborators so priviledged tests (like UI tests) can run on PRs
+  pullRequests: collaborators
 tasks:
-  $let:
-    decision_task_id: {$eval: as_slugid("decision_task")}
-    expires_in: {$fromNow: '1 year'}
-    scheduler_id: taskcluster-github
-
-    # We define the following variable at the very top, because they are used in the
-    # default definition
-    head_branch:
-      $if: 'tasks_for == "github-pull-request"'
-      then: ${event.pull_request.head.ref}
-      else:
-        $if: 'tasks_for == "github-push"'
-        then: ${event.ref}
-        else: ${event.release.target_commitish}
-
-    head_rev:
-      $if: 'tasks_for == "github-pull-request"'
-      then: ${event.pull_request.head.sha}
-      else:
-        $if: 'tasks_for == "github-push"'
-        then: ${event.after}
-        else: ${event.release.tag_name}
-
-    repository:
-      $if: 'tasks_for == "github-pull-request"'
-      then: ${event.pull_request.head.repo.html_url}
-      else: ${event.repository.html_url}
-
-    is_repo_trusted:
-      # Pull requests on main repository can't be trusted because anybody can open a PR on it, without a review
-      $if: 'tasks_for in ["github-push", "github-release"] && event.repository.html_url == "https://github.com/mozilla/application-services"'
-      then: true
-      else: false
-
-    user:
-      # dependabot-preview[bot]@users.noreply.github.com doesn't validate as email.
-      # It would be easier if TC simply didn't enforce an email format for "owner".
-      $if: 'event.sender.login == "dependabot-preview[bot]"'
-      then: dependabot
-      else: ${event.sender.login}
-  in:
-    $let:
-      images_worker_type:
-        $if: 'is_repo_trusted'
-        then: app-services-3-images
-        else: app-services-1-images
-      decision_worker_type:
-        $if: 'is_repo_trusted'
-        then: app-services-3-decision
-        else: app-services-1-decision
-      build_worker_type:
-        $if: 'is_repo_trusted'
-        then: app-services-3-b-linux
-        else: app-services-1-b-linux
-      # TODO: revisit once bug 1533314 is done to possibly infer better priorities
-      tasks_priority: highest
+  - $let:
+      taskgraph:
+        branch: taskgraph-try
+        revision: 92818b07522233961a3f91924a65e7a004028dac
```

Comment:
@mitchhentges ⚠️ Please update this revision to `d302e95cc988706ba746bfecbf1dae31090d212f` or newer. See https://github.com/mozilla-mobile/reference-browser/pull/946#issue-340586165 for more details

Reply:
I'm currently using a custom `try-push` of `taskgraph`, but I'll rebase :ok_hand: Thanks!